mods/cnc/maps/gdi02/gdi02.lua

```diff
@@ -80,7 +80,7 @@ WorldLoaded = function()
 
 	nodObjective = enemy.AddPrimaryObjective("Destroy all GDI troops.")
 	gdiObjective1 = player.AddPrimaryObjective("Eliminate all Nod forces in the area.")
-	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium Refinery.")
+	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium refinery.")
```

Comment:
Tiberium is a proper noun, thus it should be capitalized.

On 2015-06-19, at 4:53 PM, abcdefg30 <notifications@github.com> wrote:

> In mods/cnc/maps/gdi02/gdi02.lua:
> 
> > @@ -80,7 +80,7 @@ WorldLoaded = function()
> >  
> >  	nodObjective = enemy.AddPrimaryObjective("Destroy all GDI troops.")
> >  	gdiObjective1 = player.AddPrimaryObjective("Eliminate all Nod forces in the area.")
> > -	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium Refinery.")
> > +	gdiObjective2 = player.AddSecondaryObjective("Capture the Tiberium refinery.")
> Hm, okay.
> 
> ―
> Reply to this email directly or view it on GitHub.
>